src/SFA.DAS.AssessorService.Application/Helpers/FinancialReviewStatusHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using SFA.DAS.AssessorService.Api.Types.Models.AO;
+using SFA.DAS.AssessorService.Application.Interfaces;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.AssessorService.Application.Helpers
+{
+    public static class FinancialReviewStatusHelper
```

Comment:
Could we get some unit tests around confirming the behaviour of when you are financially exempt?